lib/open_mic.rb

```diff
@@ -0,0 +1,27 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+
+  def initialize(hash)
+    @location = hash[:location]
+    @date = hash[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
```

Comment:
I like the attempt here with the hash! You're thinking is on point, but the implementation is just a bit short.